horace_core/sqw/@axes_block/private/rebin_data_.m

```diff
@@ -5,7 +5,7 @@
 % according to realigh_axes method of this axes block
 
 
-data_nodes = obj.get_bin_nodes('-density_integr');
+data_nodes = obj.get_bin_nodes('-bin_center');
```

Comment:
checking how you want to be consistent on american spelling. You have center here but centre in comments further down

Reply:
actually globally renamed to `-bin_centre` everywhere